magpie/alembic/versions/03b54feffe45_ensure_anonymous_group_applied_to_users.py

```diff
@@ -5,8 +5,7 @@
 Revises: 73b872478d87
 Create Date: 2019-08-23 18:08:07.507556
 """
-# noinspection PyUnresolvedReferences
-from alembic.context import get_context
+from alembic.context import get_context  # noqa: F401
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'alembic.context'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)